python/lsst/ts/ATDome/mock_controller.py

```diff
@@ -433,8 +356,8 @@ def set_cmd_az(self, cmd_az):
         cmd_az : `astropy.coordinates.Angle`
             Desired azimuth. Must be in the range [0, 360] deg.
         """
-        self.az_actuator.set_pos(cmd_az)
-        self.last_rot_right = True if self.az_actuator.direction == 1 else False
+        self.az_actuator.set_position(cmd_az)
+        self.last_rot_right = True if self.az_actuator.direction_factor == 1 else False
```

Comment:
Same suggestion, check if you keep `direction_factor` or change it back to `direction`.

Reply:
Thank you. I removed the rename, leaving`direction`